loki/backend/pygen.py

```diff
@@ -233,9 +236,9 @@ def visit_Loop(self, o, **kwargs):
         else:
             cntrl = f'range({start}, {end} + 1)'
         header = self.format_line('for ', var, ' in ', cntrl, ':')
-        self.depth += 1
+        self.depth += self.style.indent_default
         body = self.visit(o.body, **kwargs)
-        self.depth -= 1
+        self.depth -= self.style.indent_default
         return self.join_lines(header, body)
 
     def visit_WhileLoop(self, o, **kwargs):
```

Comment:
Because codecov flags them, I'll point out a few coverage omissions in case you'd like to do some testing TLC.
This visitor method isn't tested.